src/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.BitsBuffer.cs

```diff
@@ -92,6 +95,7 @@ public void Reduce(ref FastReducer reducer)
             {
                 // Executes a modulo operation using an optimized reducer.
                 // Thus, no need of any switching here, happens in-line.
+
```

Comment:
I see.  Ok.  Some of them to me personally seem like they'd be better without the blank line, but that's just a matter of style.  What you have is fine. Thanks.